seolondon/settings/__init__.py

```diff
@@ -95,6 +95,7 @@
     'django.middleware.csrf.CsrfViewMiddleware',
     'django.contrib.auth.middleware.AuthenticationMiddleware',
     'django.contrib.messages.middleware.MessageMiddleware',
+    'django.contrib.sites.middleware.CurrentSiteMiddleware',
```

Comment:
no worries! :smile:

I want to add the full url.  In the template `request.path` will give you the part without the host, e.g. `/career/`, and the middleware allow us to use `request.site` to add the host/site part for prefixing such as `https://seo-london.org/career/`. 
https://docs.djangoproject.com/en/1.10/ref/contrib/sites/#site-middleware

Although in a second thought, facebook might not actually require the url to be full url, and we might not actually need it.... I will see if there are docs on that first :slightly_smiling_face: